core/src/test/scala/unit/kafka/admin/TopicCommandWithAdminClientTest.scala

```diff
@@ -65,9 +65,10 @@ class TopicCommandWithAdminClientTest extends KafkaServerTestHarness with Loggin
   private val numPartitions = 1
```

Comment:
We can rename this test to be `TopicCommandTest`.

Reply:
@ijuma , we already have a `TopicCommandTest` to do Unit test. This `TopicCommandWithAdminClientTest` is much like an integration test. How about rename to `TopicCommandIntegrationTest` just like `DeleteOffsetsConsumerGroupCommandIntegrationTest` in the same folder?

Reply:
So, I rename the test to `TopicCommandIntegrationTest`, and move this test to integration test folder: `core/src/test/scala/integration/kafka/admin`. Thanks.

Reply:
The name sounds good, but I think you don't have to move the test. We rely on tags instead of directory structure for distinguishing test types.

Reply:
I see. Moved back now. Thanks.